conformance/base/manifests.yaml

```diff
@@ -93,7 +93,7 @@ spec:
     spec:
       containers:
       - name: infra-backend-v1
-        image: k8s.gcr.io/ingressconformance/echoserver:v0.0.1
+        image: gcr.io/k8s-staging-ingressconformance/echoserver:v20220815-e21d1a4
```

Comment:
Where is this image defined? Can we add some kind of comment or docs if we ever need to patch it?

Reply:
https://github.com/kubernetes-sigs/ingress-controller-conformance/tree/master/images/echoserver, I will comment

Reply:
Yes, a comment to show where it originates is very appreciated :+1: